NGitLab/Models/User.cs

```diff
@@ -77,5 +107,48 @@ public class User
         [Obsolete("This does not match GitLab's Api. Use Identities.ExternUid instead.")]
         [DataMember(Name = "extern_uid")]
         public string ExternUid;
+
+        [Obsolete("This does not match GitLab's Api. Use State instead")]
```

Comment:
Why is this member obsolete?

Reply:
Doesn't appear in JSON, Gitlab API V4 checks if user is blocked with "state"  attribute

Reply:
Examples:

![image](https://user-images.githubusercontent.com/38387367/130250012-8b5d63a2-3fb3-46eb-a96d-f1ed2aa9fcdf.png)
![image](https://user-images.githubusercontent.com/38387367/130250037-7b2da075-5a44-458a-943e-1c748e9a02ba.png)

Reply:
I see. We don't have a clear way to indicate a member exists in specific versions of GitLab. We still have users on 13.x, so adding obsolete is not a great way to do it.
I prefer you remove it from this PR, and we'll think about a better way of obsoleting members later.

Reply:
Made a new commit, removing the obsolete